ex02.ex

```diff
@@ -53,7 +53,10 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
+  first2equal = fn
```

Comment:
## -1

I'm being a little picky, but this implementation will accept any two values and return false.

It would be better to have one of:

~~~ elixir
  first2equal = fn
          [ a, a | _ ]              -> true
          other when is_list(other) -> false
        end

# or…
  first2equal = fn
                  [ a, b | _ ]  -> a == b
                end
~~~